packages/core/admin/admin/src/pages/SettingsPage/pages/Webhooks/ListView/index.js

```diff
@@ -473,8 +387,8 @@ const ListView = () => {
       </Main>
       <ConfirmDialog
         isOpen={showModal}
```

Comment:
I'm not sure here, but could you say `isOpen={!!webhooksToDelete}`? This way you could eliminate the double bookkeeping with `showModal` too.

Reply:
I dont think we can do that as webhooksToDelete is modified when you select rows as well as when you click a rows delete button